students/JamesTraub/Lesson02/grid_printer2.py

```diff
@@ -0,0 +1,19 @@
+# chose to divide n by 2 to give columns an even width
+
+def print_grid(n,s):
+    pls = '+'
+    mns = '-'
+    pl = '|'
+    spc = ' '
+    dventry = s
+    grdsz = (dventry * mns)
```

Comment:
I think this might be a C lang convention to use var names with no vowels?

either way it is more readable to have `grid_size` than `grdsz` 🙂 (I'm picking on `grdsz` but it is a general comment.)

sometimes it makes sense to have short var names, and sometimes it makes sense to spell it out.